webapi/cache.go

```diff
@@ -83,13 +85,23 @@ func updateCache(ctx context.Context, db *database.VspDatabase,
 		return errors.New("dcr node reports a network ticket pool size of zero")
 	}
 
+	clients, failedConnections := wallets.Clients(ctx, cfg.NetParams)
+	if len(clients) == 0 {
+		return errors.New("Could not connect to any wallets")
```

Comment:
We should keep the error log if `len(failedConnections) > 0`, but I see no reason to return an error if `len(clients) == 0`. We aren't actually using the wallet clients for anything at this time, just checking if they are online, and we still want to update the cached information.

Reply:
Thanks for the review, fixed this in the latest fp.